### PR TITLE
Clinic

### PR DESCRIPTION
**Pull request by Draxeytina**
# Summary 📒

clinic_database

- [x] Create a file named schema_based_on_diagram.sql and implement the database from the diagram.
- [x] Join tables from many-to-many relationships.
- [x] Add the FK indexes.

### CODE REVIEW

File: clinic/schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+CREATE TABLE treatment_history_link (
+medical_history_id INT,
+treatments_id INT,
+FOREIGN KEY (medical_history_id) REFERENCES medical_histories(id),
+FOREIGN KEY (treatments_id) REFERENCES treatments (id)
+);
+
+--Create indexes for the join table
+CREATE INDEX medical_history_id_asc ON treatment_history_link(medical_history_id ASC)
+CREATE INDEX treatments_id_asc ON treatment_history_link(treatments_id ASC)
```
Comment (Munsa1) [inline clinic/schema_based_on_diagram.sql:64]: - [ ] It's a requirement that all FOREIGN KEYS have indexes created. Kindly do so:
- Check the image below for the missing indexes
image

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Draxeytina, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: clinic/schema_based_on_diagram.sql (hunk 1 above)
Comment (Draxeytina) [reply clinic/schema_based_on_diagram.sql:64]: Thank you @Munsa1 we implemented the changes.

**Review by t-yanick — APPROVED** [review]
Hi @Draxeytina, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**